first-analysis-steps/analysis-productions.md

```diff
@@ -90,7 +86,7 @@ The next file needed is a `.yaml` file, which will be used to configure the jobs
 
 ```yaml
 defaults:
-    application: DaVinci/v45r5
+    application: DaVinci/v45r8
```

Comment:
Do people think it would be worth it to update the [demo production](https://gitlab.cern.ch/lhcb-datapkg/AnalysisProductions/-/blob/master/D02HH_Starterkit/info.yaml) to use DaVinci v45r8 as well, so that it matches what's shown here?

Reply:
Yes I suppose so. I'm trying to think if there is a way to update a production without it automatically submitting a new version since it seems a bit wasteful to run jobs that nobody will actually use. But then again it only takes up ~190MB so in the grand scheme of things it's probably fine to just take the tiny loss on storage and not waste time trying to find some trick to avoid it. I guess there is also something to be said for the tuples/logs in storage to match exactly the code version used to produce them even if it's unlikely anybody will ever look at them closely.

Reply:
I would say re-running is worthwhile. It's good a good test to make sure it's working and the resoources involved are negligible.